tests/UnitTests/CoreGraphics.drawing/DrawingTest.cpp

```diff
@@ -26,6 +26,21 @@
 #import "CGContextInternal.h"
 #endif
 
+#ifdef TARGET_OS_MAC
+#define LOG_TEST_FILE(...)
```

Comment:
fair. Would be useful to move this macro to TestFramework for the benefit of other tests?

---
In reply to: [100652177](https://github.com/Microsoft/WinObjC/pull/1948#discussion_r100652177) [](ancestors = 100652177)